<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>569</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>OpenSimwheel Commander</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources/Images.qrc">
    <normaloff>:/logo/logo-glyph-48x48-transparent.ico</normaloff>:/logo/logo-glyph-48x48-transparent.ico</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>581</width>
      <height>135</height>
     </rect>
    </property>
    <property name="title">
     <string>Plugin</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Active Plugin:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Settings</string>
       </property>
       <layout class="QFormLayout" name="formLayout_5">
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Activate 1:1 mapping</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkBox_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Use Hardstops</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QGroupBox" name="groupBox_6">
       <property name="title">
        <string>Status</string>
       </property>
       <layout class="QFormLayout" name="formLayout_6">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Status:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>disconnected</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QComboBox" name="comboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>iRacing</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Assetto Corsa</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>rFactor 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>rFactor 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Richard Burns Rallye</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QDockWidget" name="vjoy_dockWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>271</width>
      <height>311</height>
     </rect>
    </property>
    <property name="floating">
     <bool>true</bool>
    </property>
    <property name="features">
     <set>QDockWidget::AllDockWidgetFeatures</set>
    </property>
    <property name="allowedAreas">
     <set>Qt::NoDockWidgetArea</set>
    </property>
    <property name="windowTitle">
     <string>vJoy Info</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents">
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>261</width>
        <height>271</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>vJoy</string>
      </property>
      <property name="title">
       <string>vJoy</string>
      </property>
      <widget class="QGroupBox" name="groupBox_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>241</width>
         <height>111</height>
        </rect>
       </property>
       <property name="title">
        <string>Driver Info</string>
       </property>
       <layout class="QFormLayout" name="formLayout_7">
        <item row="0" column="0">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>Enabled: </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="lbl_vjoy_enabled">
          <property name="text">
           <string>false</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>Vendor:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="lbl_vjoy_vendor">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>Product:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="lbl_vjoy_product">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>Version Number:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="lbl_vjoy_version">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>241</width>
         <height>122</height>
        </rect>
       </property>
       <property name="title">
        <string>Device</string>
       </property>
       <layout class="QFormLayout" name="formLayout_8">
        <item row="0" column="0">
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>ID:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="lbl_vjoy_device_id">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>Status:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="lbl_vjoy_device_status">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>Axis (min):</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="lbl_vjoy_axis_min">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string>Axis (max):</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="lbl_vjoy_axis_max">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_29">
          <property name="text">
           <string>Current Position:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="lbl_vjoy_axis_pos">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QDockWidget" name="telemetryOutput_dockWidget">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>190</y>
      <width>248</width>
      <height>162</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="floating">
     <bool>true</bool>
    </property>
    <property name="features">
     <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetMovable</set>
    </property>
    <property name="allowedAreas">
     <set>Qt::NoDockWidgetArea</set>
    </property>
    <property name="windowTitle">
     <string>Telemetry Output</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents_6">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Telemetry</string>
        </property>
        <layout class="QFormLayout" name="formLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Steering Shaft Torque</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="lbl_telemetry_torque">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>TorquePct:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="lbl_telemetry_torquePct">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>Damping:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="lbl_telemetry_damper">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Angle:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="lbl_telemetry_angle">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>Car LockToLock:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="lbl_telemetry_lockToLock">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QDockWidget" name="debugInformation_dockWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>192</y>
      <width>191</width>
      <height>251</height>
     </rect>
    </property>
    <property name="floating">
     <bool>true</bool>
    </property>
    <property name="features">
     <set>QDockWidget::AllDockWidgetFeatures</set>
    </property>
    <property name="allowedAreas">
     <set>Qt::NoDockWidgetArea</set>
    </property>
    <property name="windowTitle">
     <string>Debug Information</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents_5">
     <layout class="QFormLayout" name="formLayout_3">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="1" column="1">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Loop Debug</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Torque Setpoint: </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="lbl_torque">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Position:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="lbl_position">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Calculation Benchmark:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="lbl_calculation_benchmark">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Calc (min.):</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLabel" name="lbl_calc_min">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Calc (max.):</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QLabel" name="lbl_calc_max">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Calc (avg.):</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QLabel" name="lbl_calc_avg">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QLabel" name="lbl_lastLoopTime">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Last Loop Time:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="lbl_wheel_center_offset">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Offset:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
        <zorder>lbl_calc_avg</zorder>
        <zorder>label</zorder>
        <zorder>lbl_calc_max</zorder>
        <zorder>lbl_calc_min</zorder>
        <zorder>lbl_lastLoopTime</zorder>
        <zorder>label_6</zorder>
        <zorder>label_5</zorder>
        <zorder>label_4</zorder>
        <zorder>label_2</zorder>
        <zorder>lbl_torque</zorder>
        <zorder>label_8</zorder>
        <zorder>label_7</zorder>
        <zorder>lbl_calculation_benchmark</zorder>
        <zorder>lbl_position</zorder>
        <zorder>label_3</zorder>
        <zorder>lbl_wheel_center_offset</zorder>
        <zorder>line</zorder>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>745</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Restart"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="action_Drive_Stage"/>
    <addaction name="action_Joystick_Settings"/>
    <addaction name="separator"/>
    <addaction name="actionWheel_Settings"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Settings"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Drive_Stage">
   <property name="text">
    <string>&amp;Drive Stage Settings...</string>
   </property>
  </action>
  <action name="actionWheel_Settings">
   <property name="text">
    <string>Wheel Settings...</string>
   </property>
  </action>
  <action name="action_Debug_Information">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Debug Information</string>
   </property>
  </action>
  <action name="action_Telemetry_Output">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Telemetry Output</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About OpenSimwheel Commander...</string>
   </property>
  </action>
  <action name="action_Joystick_Settings">
   <property name="text">
    <string>&amp;Joystick Settings...</string>
   </property>
  </action>
  <action name="action_Restart">
   <property name="text">
    <string>&amp;Restart</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources/Images.qrc"/>
 </resources>
 <connections/>
</ui>
